e2e-tests/ground-helpers.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Nit: Ok to omit "Ground" from the filename since the code is already qualified by the "ground" repo. Perhaps `test-utils.ts` would work?

Reply:
Done.